system/jlib/jdebug.cpp

```diff
@@ -1096,7 +1096,11 @@ void getCpuInfo(unsigned &numCPUs, unsigned &CPUSpeed)
 static unsigned evalAffinityCpus()
 {
     unsigned numCpus = 0;
+#ifdef __64BIT__
+    DWORD64 ProcessAffinityMask, SystemAffinityMask;
```

Comment:
This is causing a memory corruption in 64bit - which was causing a very strange crash on exit (and probably a long pause on release builds).  Should possibly be backported even further.

Reply:
If you want it backported further (and probably even if you don't) then it should be in a separate JIRA/PR